application/views/admin/super/fullpagebar_view.php

```diff
@@ -17,7 +17,8 @@
                 <!-- Install Plugin Zip -->
                 <?php if (
                     isset($fullpagebar['pluginManager']['buttons']['installPluginZipModal']['hasConfigDemoMode']) &&
-                    !$fullpagebar['pluginManager']['buttons']['installPluginZipModal']['hasConfigDemoMode']
+                    !$fullpagebar['pluginManager']['buttons']['installPluginZipModal']['hasConfigDemoMode'] &&
+                    !Yii::app()->getConfig('disablePluginUpload')
```

Comment:
Adding a boolean `shownUpload` in controller maybe ? 

PS : i really think this isset is directy …

Reply:
You mean for not showing the button at all if uploads are disabled?

Reply:
Thank you